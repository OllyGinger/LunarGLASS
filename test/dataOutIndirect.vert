#version 130

attribute vec4 color;
varying vec4 colorOut[6];

void main()
{
    for (int i = 1; i < 5; ++i)
        colorOut[i] = color;

    gl_Position = colorOut[2];
}
