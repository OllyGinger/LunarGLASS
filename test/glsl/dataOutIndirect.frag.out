
***Unsupported functionality: unsupported array dereference
compiling dataOutIndirect.frag...
varying vec4 Color ; 
uniform int i ; 
void  main (){
gl_FragData [ i ] = Color ; }



(
(declare (uniform ) int i)
(declare (in ) vec4 Color)
(declare (out ) (array vec4 3) gl_FragData)
(function main
  (signature void
    (parameters
    )
    (
      (assign (constant bool (1)) (xyzw) (array_ref (var_ref gl_FragData) (var_ref i) )  (var_ref Color) ) 
    ))

)


)